python/mxnet/gluon/contrib/estimator/estimator.py

```diff
@@ -57,87 +60,85 @@ def __init__(self, net,
                  context=None):
 
         self.net = net
+        self.loss = self._check_loss(loss)
+        self.train_metrics = self._check_metrics(metrics)
+
+        # Use default mx.metric.Accuracy() for gluon.loss.SoftmaxCrossEntropyLoss()
+        if not self.train_metrics and any([isinstance(l, gluon.loss.SoftmaxCrossEntropyLoss) for l in self.loss]):
```

Comment:
instead of hard coding it here, how about specify it as a default argument?

Reply:
@szha  Agree, but it Acc as default may not work well for other type of loss. How about moving this logic to prepare_loss_and_metrics? So the reference is returned to user.